tests/private-active-active/main.tf

```diff
@@ -56,7 +60,7 @@ module "private_active_active" {
   tfe_license_filepath        = ""
   tfe_subdomain               = "${local.friendly_name_prefix}-test-private-active-active"
 
-  common_tags = local.common_tags
+  asg_tags = local.common_tags
 }
 
 data "aws_instances" "main" {
```

Comment:
The filter logic for this data source is now insufficient as the internal proxy instance has the tags but not the key needed to authorize our SOCKS5 proxy connection. This isn't an issue for private-tcp-active-active as it was already designed to use the internal proxy instance for the SOCKS5 proxy as well: https://github.com/hashicorp/terraform-aws-terraform-enterprise/blob/annie-use-default-tagging/tests/private-tcp-active-active/outputs.tf#L34. We should update this case to do the same (and probably update the output names on both tests to be `proxy_instance_id`.